tailcfg/derpmap.go

```diff
@@ -144,5 +155,14 @@ type DERPNode struct {
 	STUNTestIP string `json:",omitempty"`
 }
 
+// DERPLocation contains information about a DERP region's physical location.
+type DERPLocation struct {
```

Comment:
The geohash locations we use in https://mon.corp.ts.net/d/lBFZ0P_Wk/derp?orgId=1 are the center of the city where the datacenter is located, which will probably be fine.

Those geohashes are supplied as metadata in the prometheus.yml.

Reply:
Ooooh, yeah, let's use a Geohash string omitempty instead of a new type.

Plus geohash was made by somebody pretty active in the Go community :)

Reply:
Is there any reason to use a geohash on the wire? It's one thing to show that to a user, but you'd have to turn them into a lat/lon pair to do distance calculations anyway, so I don't think it really gains us anything. Lat/lon is pretty unambiguous, IMO.